v3/src/data-interactive/handlers/attribute-handler.ts

```diff
@@ -74,96 +109,17 @@ export const diAttributeHandler: DIHandler = {
     }
     return attributeNotFoundResult
   },
-  delete: diNotImplementedYet
+  delete(resources: DIResources) {
+    const { attribute, dataContext } = resources
+    if (!attribute) return attributeNotFoundResult
+    if (!dataContext) return dataContextNotFoundResult
+
+    dataContext.applyUndoableAction(() => {
+      withoutUndo()
```

Comment:
Ok for now, but this is the kind of operation that should either be undoable or should clear the undo history entirely because attempting to undo past an unrecorded attribute deletion can lead to hard crashes.